### PR TITLE
Threepio integration

### PR DESCRIPTION
**Pull request by Nolski**
This patch switches us to using Threepio in order to perform command translation.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -59,6 +59,7 @@
     }
   },
   "dependencies": {
+    "@openmined/threepio": "github:openmined/threepio",
```
Comment (vvmnnnkv) [inline package.json:62]: cool :)

File: src/_helpers.js [hunk 2]
```diff
@@ -24,35 +24,7 @@ export const pickTensors = tree => {
...
-  command = command.split('torch.').join('');
-
-  // If the command as it's currently named exists in TensorFlow.js already, return the command name
-  if (Object.hasOwnProperty.call(tf, command)) return command;
-
-  // If not, we will need to do a lookup of the command in question
-  logger.log(CANNOT_FIND_COMMAND(command));
-
-  return null;
+export const torchToTF = command => {
```
Comment (vvmnnnkv) [inline src/_helpers.js]: I'd just remove that helper function and placed
`const threepio = new Threepio('torch', 'tfjs', tf);`
on top of message.js, so we init it only once.

File: src/types/message.js [hunk 3]
```diff
@@ -92,37 +93,34 @@ export class Operation extends Message {
...
+        // Get the actual tensor inside the PointerTensor/Placeholder "this.owner" and place it in front of args
+        args.unshift(getTensorByRef(this.owner));
 
-        // Now we can execute a multi-argument method
-        result = tf[command](self, ...preArgs, ...args, ...postArgs);
+        const translation = torchToTF(
+          new Command(functionName, args, this.kwargs)
+        );
+        translation.args = pullTensorsFromArgs(translation.args);
+        console.log(translation);
```
Comment (vvmnnnkv) [inline src/types/message.js]: Please remove debug

File: test/_helpers.test.js [hunk 4]
```diff
@@ -22,15 +20,8 @@ describe('Helpers', () => {
     expect(Object.keys(objects).length).toBe(2);
   });
 
-  test('torchToTF(): can convert underscore function', () => {
-    expect(torchToTF('__add__', {}, logger)).toBe('add');
-  });
-
   test('torchToTF(): can convert torch function', () => {
```
Comment (vvmnnnkv) [inline test/_helpers.test.js]: That can be dropped together with torchToTf helper function.
We don't need to test what's already tested in threepio itself :)

**Review by vvmnnnkv — CHANGES_REQUESTED** [review]

**Review by cereallarceny — APPROVED** [review]

**Pull request merged**